output/transform/postgres_buffer_cache.go

```diff
@@ -0,0 +1,22 @@
+package transform
+
+import (
+	snapshot "github.com/pganalyze/collector/output/pganalyze_collector"
+	"github.com/pganalyze/collector/state"
+)
+
+func transformPostgresBufferCache(s snapshot.FullSnapshot, ts state.TransientState, databaseOidToIdx OidToIdx) snapshot.FullSnapshot {
+	for databaseOid, bufferCache := range ts.BufferCache {
+		databaseIdx, ok := databaseOidToIdx[databaseOid]
+		if ok {
+			var untrackedBytes int64
+			for _, bytes := range bufferCache {
+				untrackedBytes += bytes
+				continue
+			}
+			s.DatabaseStatictics[databaseIdx].UntrackedCacheBytes = untrackedBytes
```

Comment:
I don't have an intended use for this data, but it could be useful if not all databases / tables are being monitored by the collector.

Note: `input/postgres/relations.go` calls `bufferCache[dataFilenode] = 0` to zero out buffer cache entries, so that the code here only tracks buffer usage not associated with a known table/index.